es5-shim.js

```diff
@@ -1654,7 +1764,7 @@ defineProperties(StringPrototype, {
     }
 }, hasTrimWhitespaceBug);
 
-var hasLastIndexBug = StringPrototype.lastIndexOf && 'abcあい'.lastIndexOf('あい', 2) !== -1;
+var hasLastIndexBug = StringPrototype.lastIndexOf && 'abc\u3042\u3044'.lastIndexOf('\u3042\u3044', 2) !== -1;
```

Comment:
Thanks - It's only an issue if text encoding isn't properly set up - I'm not concerned about it at this point.